tests/lib/rules/no-broken-super-chain.js

```diff
@@ -104,32 +169,36 @@ ruleTester.run('no-broken-super-chain', rule, {
           }
         });`,
       errors: [{
-        message: noSuper
+        message: `${noSuper} init`
       }]
     },
-    // TODO
-    // {
-    //   code: `
-    //     export default Ember.Component.extend({
-    //       init() {
-    //         this._super(); // missing '...arguments'
-    //         this.alias = this.concrete;
-    //       }
-    //     });`,
-    // },
-    // TODO
-    // {
-    //   code: `
-    //     export default Ember.Component.extend({
-    //       init() {
-    //         this.alias = this.concrete;
-    //         this._super(...arguments);
-    //       }
-    //     });`,
-    //   errors: [{
-    //     message: noThisBeforeSuper
-    //   }]
-    // },
+    {
+      code: `
+        export default Ember.Component.extend({
+          init() {
+            this._super(); // missing '...arguments'
+            this.alias = this.concrete;
+          }
+        });`,
+      errors: [{
+        message: argsNotPassedToSuper
+      }]
+    },
+    /**
+    TODO
+    {
+      code: `
+        export default Ember.Component.extend({
+          init() {
+            this.alias = this.concrete;
+            this._super(...arguments);
+          }
+        });`,
+      errors: [{
+        message: noThisBeforeSuper
```

Comment:
Let’s remove this TODO commented test.

Reply:
When we add a rule about no this before super we will probably make it a new more focused rule anyways.

Reply:
Removed TODO